testnet/install.rst

```diff
@@ -22,19 +22,29 @@ For that, there are 5 steps to follow:
   * `Windows <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_windows_amd64.exe>`_
   * MacOS
 
-    * `ARM <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_darwin_arm64>`_
-    * `AMD <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_darwin_amd64>`_
+    * For Apple Silicon CPUs (M1, M2...),
+      use `ARM <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_darwin_arm64.tar.gz>`_
 
-  * `Linux <https://github.com/massalabs/thyra/releases/latest/download/thyra-server_linux_amd64>`_
+    * For Intel CPUs (i5, i7...),
+      use `AMD <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_darwin_amd64.tar.gz>`_
+
+  * `Linux <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_linux_amd64.tar.gz>`_
 
 2. Start thyra using the icon tray.
-   If you need help, this `page <https://github.com/massalabs/thyra/blob/main/INSTALLATION.md>`_ can help.
-3. Configure your VPS (if not done already) and go to `thyra home page <https://my.massa/thyra/home/>`_
+
+  If you need help, this `page <https://github.com/massalabs/thyra/blob/main/INSTALLATION.md>`_ can help.
+
+3. `Configure your VPS <https://github.com/massalabs/thyra-node-manager-plugin/wiki>`_
+
+  (if not done already) and go to `thyra home page <https://my.massa/thyra/home/>`_
+
```

Comment:
@qdrn because of the linter that limit the length of the line we are force to go to the line bellow and so the render is a bit spaced

Reply:
Changed, should be ok now.